packages/orchestrator-ui-components/src/components/WfoSubscription/WfoSubscriptionProductBlock/WfoSubscriptionProductBlock.tsx

```diff
@@ -113,16 +116,42 @@ export const WfoSubscriptionProductBlock = ({
                         <tbody>
                             {isOutsideCurrentSubscription && (
                                 <tr key={-1} css={rowStyle}>
-                                    <td css={leftColumnStyle}>
+                                    <td
+                                        css={leftColumnStyle}
+                                        style={{ alignSelf: 'center' }}
```

Comment:
style and css together might be confusing, maybe add `{ alignSelf: 'center' }` to the `leftColumnStyle`

Reply:
It needs to be on this column only though and I didn't wan to make a separate leftColumnStyleWithAlignSelf style but ok I'll do that if this causes confusion